src/http/achievements/mod.rs

```diff
@@ -20,8 +20,7 @@ mod payload;
 #[post("/", format = "json", data = "<achievement_create_req>")]
 async fn add_achievement(
     state: &State<MarsAPIState>,
-    achievement_create_req: Json<AchievementCreateRequest>,
-    _auth_guard: AuthorizationToken
```

Comment:
This change removes the auth guard, we need to keep this unless there's a reason for this